cfme/tests/test_rest.py

```diff
@@ -764,6 +830,141 @@ def _finished():
     wait_for(_finished, num_sec=600, delay=5, message="REST automation_request finishes")
 
 
+@pytest.mark.ignore_stream("5.4")
+@pytest.mark.parametrize(
+    "multiple", [False, True],
+    ids=["one_request", "multiple_requests"])
+def test_edit_categories(rest_api, categories, multiple):
+    if "edit" not in rest_api.collections.categories.action.all:
+        pytest.skip("Edit categories action is not implemented in this version")
+
+    if multiple:
+        new_names = []
+        ctgs_data_edited = []
+        for ctg in categories:
+            new_name = fauxfactory.gen_alphanumeric().lower()
+            new_names.append(new_name)
+            ctg.reload()
+            ctgs_data_edited.append({
+                "href": ctg.href,
+                "description": "test_category_{}".format(new_name),
+            })
+        rest_api.collections.categories.action.edit(*ctgs_data_edited)
+        for new_name in new_names:
+            wait_for(
+                lambda: rest_api.collections.categories.find_by(description=new_name),
+                num_sec=180,
+                delay=10,
+            )
+    else:
+        ctg = rest_api.collections.categories.find_by(description=categories[0].description)[0]
+        new_name = 'test_category_{}'.format(fauxfactory.gen_alphanumeric().lower())
+        ctg.action.edit(description=new_name)
+        wait_for(
+            lambda: rest_api.collections.categories.find_by(description=new_name),
+            num_sec=180,
+            delay=10,
+        )
+
+
+@pytest.mark.ignore_stream("5.4")
```

Comment:
Looks like we got the ignore_stream/uncollect on this one. Personally I prefer uncollectif because we can specify that anything below a certain value should be uncollected, not a specific stream.